From 5a0181e5deb820399f7034b735fccbf33ad27f54 Mon Sep 17 00:00:00 2001
From: Sujeet Baranwal <sbaranwal@marvell.com>
Date: Mon, 18 Nov 2019 14:47:36 -0800
Subject: [PATCH 436/767] scmi: Increase timeout for Rx response

commit 3e66adb2865542170d3f2aad8ca8ac839eb2825f from
git@git.assembla.com:cavium/WindRiver.linux.git

After submitting a work to SCP, various platform takes different
amount of time to process the requests. In case of, octeon tx, it
takes some times more than default time. The increase ensures
adequate time for SCP to respond.

Bug: IPBUSW-2243

Change-Id: I71386c92958b2763922a1c40ea8a5a2076518ec1
Signed-off-by: Sujeet Baranwal <sbaranwal@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/19144
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/firmware/arm_scmi/driver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/firmware/arm_scmi/driver.c b/drivers/firmware/arm_scmi/driver.c
index b49c9e6f4bf1..c910ab489518 100644
--- a/drivers/firmware/arm_scmi/driver.c
+++ b/drivers/firmware/arm_scmi/driver.c
@@ -606,7 +606,7 @@ int scmi_handle_put(const struct scmi_handle *handle)
 }
 
 static const struct scmi_desc scmi_generic_desc = {
-	.max_rx_timeout_ms = 30,	/* We may increase this if required */
+	.max_rx_timeout_ms = 300,	/* We may increase this if required */
 	.max_msg = 20,		/* Limited by MBOX_TX_QUEUE_LEN */
 	.max_msg_size = 128,
 };
-- 
2.31.1

